app/assets/stylesheets/application.scss

```diff
@@ -15,10 +15,72 @@
  *= require foundation_and_overrides
  */
 
+ @import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,400italic,300italic);
+
+ @font-face {
+	font-family: 'icomoon';
+	src:url('../fonts/icomoon/icomoon.eot');
```

Comment:
Colon after property should be followed by one space<br>Prefer double-quoted strings